src/app/shared/services/auth.service.ts

```diff
@@ -67,7 +68,7 @@ export class AuthService {
       displayName: user.displayName ? user.displayName : user.email
     };
 
-    if (user.photoURL) data.photoUrl = user.photoURL;
+    data.photoUrl = user.photoURL ? user.photoURL : "../assets/images/default.jpg";
```

Comment:
Url should be a valid url, not relative to the project. You can upload an image to firebase storage bucket or just use imgur.

Reply:
Fixed